backend/Models/Data/Species.cs

```diff
@@ -0,0 +1,12 @@
+namespace WhaleSpotting.Models.Data;
+
+public class Species
+{
+    public int SpeciesId { get; set; }
+    public string SpeciesName { get; set; }
+    public string ExampleLink { get; set; }
+    public string TailPictureLink { get; set; }
+    public string WikiLink { get; set; }
+    public int TotalSightings { get; set; }
```

Comment:
I think this field should either be optional or have a default value of 0 so that when it's being being populate we won't have to add any logic to cover situations where either there haven't been sightings yet, or we don't know if there have been any sightings of that species. What do you think?

Reply:
One question sorry, do you mean setting as optional also the fields ExampleLink, TailPictureLink and WikiLink?

Reply:
No, just the TotalSightings

Reply:
Ok, we'll set it to 0 by default and do a new commit. Thank you!